check-unpinned-deps.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs')
+const { join } = require('path')
+
+/**
+ * Scan dependencies in the package.json files in these folders.
+ */
+const folders = ['.', './editor', './utopia-remix', './puppeteer-tests']
```

Comment:
Can you add `utopia-api`, `utopia-vscode-common`, and `utopia-vscode-extension` to this list please